appconfiguration/feature-management-web-sample/pom.xml

```diff
@@ -28,22 +29,16 @@
       <artifactId>spring-boot-starter-thymeleaf</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-feature-management</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-feature-management;dependency} -->
+      <groupId>com.azure.spring</groupId>
+      <artifactId>azure-spring-cloud-feature-management</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>spring-cloud-azure-feature-management-web</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-azure-feature-management-web;dependency} -->
+      <groupId>com.azure.spring</groupId>
+      <artifactId>azure-spring-cloud-feature-management-web</artifactId>
```

Comment:
Does the `azure-spring-cloud-feature-management-web` already contains the dependency `azure-spring-cloud-feature-management`?